src/TimeZones.jl

```diff
@@ -76,6 +77,25 @@ function TimeZone(str::AbstractString)
     end
 end
 
+
+"""
+    @tz_str str -> TimeZone
```

Comment:
After looking up the docstring for `@r_str` it appears this should just be `@tz_str -> TimeZone`